### PR TITLE
refactor: rename bigquery source type

### PR DESCRIPTION
**Pull request by AdriSolid**
Rename BigQuery source type, from `bq` to `bigquery`

For: https://app.clubhouse.io/cartoteam/story/135148/rename-source-type-in-template

Related with: https://github.com/CartoDB/carto-react-template/pull/203

### CODE REVIEW

File: docs/api-reference/redux.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Parameters are in the form of destructuring arguments.
 | ------ | ------------------- | ----------------------------------------------------------------------------------------------------- |
 | {id}   | <code>string</code> | unique id for the source                                                                              |
 | {data} | <code>string</code> | data definition for the source. Query for SQL dataset or the name of the tileset for BigQuery Tileset |
-| {type} | <code>string</code> | type of source. Posible values are sql or bq                                                          |
+| {type} | <code>string</code> | type of source. Posible values are sql or bigquery                                                    |
```
Comment (VictorVelarde) [inline docs/api-reference/redux.md:44]: 'sql' or 'bigquery'

Comment (AdriSolid) [reply docs/api-reference/redux.md:44]: This is correct right? Am I missing something?

Comment (AdriSolid) [reply docs/api-reference/redux.md:44]: The commas?

Comment (VictorVelarde) [reply docs/api-reference/redux.md:44]: yeap,. commas

**Comment by VictorVelarde** [comment]
@AdriSolid The most important thing here is to update also the TEMPLATE project, including its hygen (that other PR I pinged you about). Please don't merge this until having the other one prepared and tested. Ping me back when you have both working please

**Review by VictorVelarde — APPROVED** [review]
👍🏻

**Review by VictorVelarde — APPROVED** [review]
👍🏻

**Pull request merged**